developer/openshift/reset.sh

```diff
@@ -106,22 +106,52 @@ prechecks() {
     export KUBECONFIG="$KUBECONFIG"
 }
 
-uinstall_minio() {
-    printf "\n  Uninstalling Minio:\n"
+check_if_argocd_app_was_removed() {
+  local appName=$1
+  local numOfAttempts=$2
+  local i=0
+  printf "    Wait while argocd application '%s' will be removed " "$appName"
+  while [[ $(argocd app get "${appName}" 2>&1) != *"not found"* ]]; do
+    printf '.'; sleep 5;
+    i=$((i+1))
+    if [[ $i -eq "${numOfAttempts}" ]]; then
+      printf "\n[ERROR] Argocd app %s was no deleted by timeout \n" "$appName"
+      printf "You can await a bit and execute script one more time. Maybe 'remove' process spends more time then expected."
+      printf "If it doens't help try to use 'reset.sh' with enabled hard reset mode: './reset.sh --work-dir /path --reset-hard true'."
```

Comment:
`If it is still failing, try to run the script again with the '--reset-hard' option.`
There's a bug in reset.sh, please to remove the `shift` for `--reset-hard` line 65, as it does not expect any parameter.